recipes/xerces-c/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+	export CC=clang
+	export CXX=clang++
```

Comment:
I wonder if using clang here makes this play nice with other stuff compiled with gcc. Not sure how to proceed.